include/clang/AST/ASTStructuralEquivalence.h

```diff
@@ -70,9 +73,11 @@ struct StructuralEquivalenceContext {
   StructuralEquivalenceContext(ASTContext &FromCtx, ASTContext &ToCtx,
                                StructuralEquivalenceKind EqKind,
                                bool StrictTypeSpelling = false,
-                               bool Complain = true)
+                               bool Complain = true,
+                               bool ErrorOnTagTypeMismatch = false)
       : FromCtx(FromCtx), ToCtx(ToCtx), EqKind(EqKind),
-        StrictTypeSpelling(StrictTypeSpelling), Complain(Complain) {}
+        StrictTypeSpelling(StrictTypeSpelling),
+        ErrorOnTagTypeMismatch(ErrorOnTagTypeMismatch), Complain(Complain) {}
```

Comment:
The `ErrorOnTagTypeMismatch` belongs to the end of the initializer list.

Reply:
Done